app/components/guide-section/subsection/component.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+
+
+export default class SubsectionComponent extends Component {
+  @service intl;
```

Comment:
Even though the class name may be long, I like to keep the names standard (what the CLI would generate).

```suggestion

export default class GuideSectionSubsectionComponent extends Component {
  @service intl;
```